json/tbd_warehouse7.json

```diff
@@ -0,0 +1,35 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing heads, sills & jambs for test_warehouse.osm",
+  "psis": [
+    {
+      "id": "detailed",
+      "head": 0.500,
+      "sill": 0.300,
+      "jamb": 0.100
```

Comment:
Users can either simply rely on a general _fenestration_ thermal bridge type (as before), or differentiate between subsurface _head, sill vs jamb_ edges.

Reply:
Do window edges derate the parent surface or the window?  Maybe it is an option?  Can the user decide to do advanced modeling of the window using https://windows.lbl.gov/software/window and tell TBD not to derate head, sill, and jambs?  Would they just put in 1.0 for the psi factors?

Reply:
Great bunch of questions grouped together (a good reminder of what's needed in user guide material). Firstly, TBD never derates subsurfaces, as per BETBG guidelines (which I quite agree with). Doing so would muddy the waters of what falls into the scope of relevant fenestration standards such as NFRC 100 and CSA A440.2, and how they're referenced in codes. These standards already factor-in the effect of frames/mullions/dividers in the calculation of the overall transmittance/conductance of the fenestrated product per se. The [frame/divider](https://openstudiocoalition.org/tutorials/tutorial_windowproperty_frameanddivider/) option (which is great) is an extension of this. What the standards (and the frame/divider option) do not offer is PSI data on how to derate host surfaces (where thermal bridging is mainly linked to the concentration of supporting structural elements around the opening e.g., lintels). Although TBD relies on similar terminology (e.g., jambs vs heads), it is only interested in the brute subsurface opening edges (shared with the parent surface). So there's no overlap.

Reply:
Side note: if one wants to cancel the thermal bridging effect of an edge in TBD, one specifies "0.000" (W/K per m) as a PSI value - not "1.000" (the latter suggests a huge thermal bridge e.g., a cantilevered balcony).

Reply:
This [statement](https://bigladdersoftware.com/epx/docs/8-2/input-output-reference/page-011.html#windowpropertyframeanddivider) "The window vertices that you specify in the FenestrationSurface:Detailed object are those of the glazed part of the window, not the frame. EnergyPlus automatically subtracts the area of the frame–determined from the glazing dimensions and the frame width–from the area of the wall containing the window." ... makes me pause. I need to dive into this rabbit hole, I'm afraid. If in some [cases](https://bigladdersoftware.com/epx/docs/8-4/input-output-reference/group-surface-construction-elements.html#windowmaterialsimpleglazingsystem) the OSM vertices are defining "outer" edges of a subsurface, while they may be defining the "inner" edges of a subsurface elsewhere, then TBD needs suitable logic/rules to identify which cases it's processing, and adapt. I'm assuming that there must be some API function that reports overall subsurface area (regardless of whether inner or outer edges are defined) - this would be required to consistently determine WWR.

Reply:
@macumber : see [here](https://github.com/rd2/tbd/pull/60#discussion_r649312578).

Reply:
The Frame & Divider (framedivider.rb) workarounds address these issues (it was worth diving into the rabbit hole). Ideally, this should eventually make its way into the OpenStudio API.